package.json

```diff
@@ -7,7 +7,6 @@
   "license": "MIT",
   "scripts": {
     "lint": "eslint . --ignore-pattern \"!.*\"",
-    "pretest": "npm run lint",
```

Comment:
why removing this?

Reply:
When I send #108, I didn't notice, run `test` also run `lint`, so I setup a separate step for `lint` https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/31951d4127f2b49cda7400a41629c05091a1ea9d/.github/workflows/main.yml#L30

So `lint` will run twice, you prefer the old way or remove the `lint` step on Github Actions?

Reply:
I see. no need to lint twice. 👍